src/controllers/initgamecontroller.js

```diff
@@ -29,6 +29,10 @@ class InitGameController extends Controller
                 sim.registerEvent();
             }
         );
+
+        $(document).ready(function () {
```

Comment:
Then this will do for now. Do remove the document ready check though, as it - AFAICT - serves no real purpose.